either/index.ts

```diff
@@ -193,11 +194,11 @@ class EitherConstructor<L, R, T extends EitherType = EitherType>
     return EitherConstructor.right(v);
   }
 
-  static fromPromise<L, R>(promise: Promise<R>): Promise<Either<L, R>> {
+  static fromPromise<R, L = unknown>(promise: Promise<R>): Promise<Either<L, R>> {
```

Comment:
Could we save the previous order of params by adding the `unknown` default value to the `R` param too?

Reply:
That would be counter intuitive since whenever working with promises most of the the time you care about the result which is now first generic argument.  From the other side this is "breaking" change.

Reply:
It's a breaking change. I will put it into the new major version